### PR TITLE
Update julia-python-and-r.md

### PR DESCRIPTION
**Pull request by dA505819**
Added tutorial.

### CODE REVIEW

**Comment by dA505819** [comment]
@neomatrix369 Sure! I restructure it.

**Comment by dA505819** [comment]
@neomatrix369 Where should I add R heading?

**Comment by neomatrix369** [comment]
> @neomatrix369 Where should I add R heading?
> Or Mani, you add it, I will commit in that heading?

A sub-section in the same page:
```
### R
- [your link]
- [your link]
```
Or something creative.

**Comment by dA505819** [comment]
@neomatrix369 Did it. Please check it.

File: details/julia-python-and-r.md [hunk 1]
```diff
@@ -206,6 +206,11 @@
 
 See [Programming in Python](../Programming-in-Python.md)
 
+## Programming in R
```
Comment (neomatrix369) [inline details/julia-python-and-r.md:209]: 👍

**Comment by neomatrix369** [comment]
Thanks @dA505819!

**Comment by dA505819** [comment]
@neomatrix369 You're Welcome!:)

**Pull request merged**